packages/shared/src/buttons/__tests__/IncrementDecrementButtons.playground.test.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { PlaygroundRenderer } from '@kiwicom/mobile-playground';
+
+import IncrementDecrementButtons from '../IncrementDecrementButtons';
+
+it('renders', () => {
+  const onIncrement = jest.fn();
+  const onDecrement = jest.fn();
+  PlaygroundRenderer.render(
+    <IncrementDecrementButtons
+      onIncrement={onIncrement}
+      onDecrement={onDecrement}
+      number={2}
+      min={1}
+      max={3}
+    />,
+    false,
+    'IncrementDecrementButtons',
+  );
+});
+
+it('shows number', () => {
+  const onIncrement = jest.fn();
+  const onDecrement = jest.fn();
+  PlaygroundRenderer.render(
+    <View style={{ backgroundColor: 'white', padding: 10 }}>
+      <IncrementDecrementButtons
+        onIncrement={onIncrement}
+        onDecrement={onDecrement}
+        number={2}
+        showNumber={true}
+        min={1}
+        max={3}
+        numberStyle={{ padding: 10 }}
+      />
+    </View>,
+    false,
+    'IncrementDecrementButtons',
+  );
+});
+
+it('render disabled', () => {
+  const onIncrement = jest.fn();
+  const onDecrement = jest.fn();
+  PlaygroundRenderer.render(
+    <View style={{ backgroundColor: 'white', padding: 10 }}>
```

Comment:
Maybe we could create a component container for these tests?

Reply:
There is probably much that could be improved on the DX of our Playground. Feel free to look at it when we have spare time 🙂

Reply:
Impossible at the moment 😄